### PR TITLE
ignite-14600 ssl test for thin client, handshake issue

### PR DESCRIPTION
**Pull request by eadha**
SSL connection test for thin client.

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: modules/ducktests/tests/ignitetest/services/utils/templates/ignite.xml.j2 [hunk 1]
```diff
@@ -80,10 +80,11 @@
         {% endif %}
 
         {% if config.ssl_params %}
-            <property name="sslContextFactory">
-                {{ ssl_params_util.ssl_params(config.ssl_params) }}
-            </property>
-        {% endif %}
+             <property name="sslContextFactory">
```
Comment (nizhikov) [inline modules/ducktests/tests/ignitetest/services/utils/templates/ignite.xml.j2:83]: Why this change?

File: modules/ducktests/tests/ignitetest/tests/thin_client_test.py [hunk 2]
```diff
@@ -32,6 +32,11 @@ class ThinClientTest(IgniteTest):
     """
     cluster - cluster size.
     JAVA_CLIENT_CLASS_NAME - running classname.
+    to use with ssl enabled:
+    export GLOBALS='{"ssl":{"enabled":true},"key_store_path":"/opt/ignite-dev/modules/ducktests/tests/certs/",
```
Comment (nizhikov) [inline modules/ducktests/tests/ignitetest/tests/thin_client_test.py:36]: Actually, we don't need anything except "ssl:enabled:true".
Please, fix the doc.

**Comment by eadha** [comment]
new one was raised 
https://github.com/apache/ignite/pull/9071/
